operations/reservations.md

```diff
@@ -661,6 +701,7 @@ Updates information about a reservation. Note that if any of the fields are sent
 | `CompanyId` | [String update value](reservations.md#string-update-value) | optional | Identifier of the [Company](enterprises.md#company) on behalf of which the reservation was made \(or `null` if company should not be updated). |
 | `BusinessSegmentId` | [String update value](reservations.md#string-update-value) | optional | Identifier of the reservation [Business segment](services.md#business-segment) \(or `null` if the business segment should not be updated).|
 | `RateId` | [String update value](reservations.md#string-update-value) | optional | Identifier of the reservation [Rate](services.md#rate) \(or `null` if the rate should not be updated). |
+| `TimeUnitPrices` | [Time unit amount update value](reservations.md#time-unit-amount-update-value) | optional | Time unit amounts. e.g amount of the first or the second night. \(or `null` if the unit amounts should not be updated). |
```

Comment:
Prices for time units of the reservation. E.g. prices for the first or the second night.